diff --git a/lib/index.d.ts b/lib/index.d.ts
index 18e6965b753ec32bbb4d6d7e7039bc66db38f3a0..3e5e0c58d18de6907310e5400f2a1d7c13d20a68 100644
--- a/lib/index.d.ts
+++ b/lib/index.d.ts
@@ -13,3 +13,4 @@ export const SugarHigh: {
   }
   TokenMap: Map<string, number>
 }
+export const Keywords_Js: Set<string>
\ No newline at end of file
diff --git a/lib/index.js b/lib/index.js
index 234a04f8b4b43ca03d196b761a59c000a4e0c441..6d8f8855c265996469dae9dd91b9b75714a7f08d 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -758,4 +758,5 @@ export {
   tokenize,
   generate,
   SugarHigh,
+  Keywords_Js,
 }
